next.config.js

```diff
@@ -7,6 +7,9 @@ const nextConfig = {
   async rewrites() {
     return [
       {
+        source: '/:path((?!main/article|domain).*)',
+        destination: `${process.env.NEXT_PUBLIC_API_URL}/:path*`,
+        // @우찬 - env파일 공유 한번 해주세요~!
         source: '/:path*',
         destination: `${
           process.env.NEXT_PUBLIC_API_MOCK === 'enabled' ? process.env.MOCK_API_URL : process.env.NEXT_PUBLIC_API_URL
```

Comment:
@dladncks1217  이 부분에서 merge하면서 conflict가 나서 일단 resolve를 임시로 해두었는데,
1. dynamic routing에 대한 처리
2. MSW mocking 서버 분기처리
두가지 모두 만족하게 처리해야할 것 같습니다

```suggestion
      {
        source: '/:path((?!main/article|domain).*)',
        destination: `${
          process.env.NEXT_PUBLIC_API_MOCK === 'enabled' ? process.env.MOCK_API_URL : process.env.NEXT_PUBLIC_API_URL
          }/:path*`,
```

이렇게 하면 괜찮을지 확인 한번 부탁해요!!

Reply:
넵넵 그렇게 해도 될것같습니다
아니면 아래처럼
```  
{
        source: 특정경로,
 destination: `${
          process.env.NEXT_PUBLIC_API_MOCK === 'enabled' ? process.env.MOCK_API_URL : process.env.NEXT_PUBLIC_API_URL
 }
 },
 {
   source: '/:path*',
        destination: `${
          process.env.NEXT_PUBLIC_API_MOCK === 'enabled' ? process.env.MOCK_API_URL : process.env.NEXT_PUBLIC_API_URL
 }
```

이렇게 따로따로 해도 괜찮을 것 같아요!
편하신대로 해주시죠